pkg/operator/mon/mon.go

```diff
@@ -116,18 +116,41 @@ func (c *Cluster) createMonSecretsAndSave(clientset *kubernetes.Clientset) (*mon
 		return nil, fmt.Errorf("failed to create mon secrets. %+v", err)
 	}
 
-	// store the secrets
+	// store the secrets for internal usage of the rook pods
 	secrets := map[string]string{
 		clusterSecretName: info.Name,
 		fsidSecretName:    info.FSID,
 		monSecretName:     info.MonitorSecret,
 		adminSecretName:   info.AdminSecret,
 	}
-	_, err = clientset.Secrets(c.Namespace).Create(&v1.Secret{ObjectMeta: v1.ObjectMeta{Name: appName}, StringData: secrets})
+	secret := &v1.Secret{
```

Comment:
does this change the way that the toolbox yaml should be accessing the admin secret? https://github.com/jbw976/rook/blob/toolbox-k8s/toolbox/rook-tools.yml#L13